src/agents/image_genai_agent.ts

```
@@ -2,6 +2,7 @@
 import { GraphAILogger } from "graphai";
 import type { AgentFunction, AgentFunctionInfo } from "graphai";
 import { provider2ImageAgent } from "../utils/provider2agent.js";
+import { apiKeyMissingError, agentGenerationError, agentInvalidResponseError, imageAction, imageFileTarget } from "../utils/error_cause.js";
```

ESLint

[error] Remove this unused import of 'agentGenerationError'.

(sonarjs/unused-import)